cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/lineage/field/FieldLineageDataset.java

```diff
@@ -441,4 +467,29 @@ private void addEndPoint(MDSKey.Builder keyBuilder, EndPoint endPoint) {
     builder.add(endPointField.getField());
     return builder.build().getKey();
   }
+
+  private boolean matchEndpoint(byte[] column, EndPoint endPoint) {
```

Comment:
nit: if a method returns a boolean, it's better to name it something that flows when saying 'if [method]'. In this case, better to name it matchesEndpoint.